ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/json/RaoParametersDeserializer.java

```diff
@@ -71,6 +71,14 @@ public RaoParameters deserialize(JsonParser parser, DeserializationContext deser
                     parser.nextToken();
                     parameters.setHvdcSensitivityThreshold(parser.getDoubleValue());
                     break;
+                case "injection-penalty-cost":
+                    parser.nextToken();
+                    parameters.setInjectionRaPenaltyCost(parser.getDoubleValue());
+                    break;
+                case "injection-sensitivity-threshold":
```

Comment:
what do you think about adding "ra" to these parameters' names in order to prevent confusion?

Reply:
right, I agree it is not so clear, I was wondering about the same thing but keep the consistency with the others "pst-penalty-cost", "hvdc-penalty-cost". If you also think it's confusion, I change it though